lib/ui/scripts/config.js

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+import webpack from 'webpack';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import UglifyJsPlugin from 'uglifyjs-webpack-plugin';
+
+const resolveLocal = dir => path.join(__dirname, dir);
+
+const r = resolveLocal('../../../node_modules');
+const out = resolveLocal('../../core/dll');
+
+export default ({ entry, provided = [] }) => ({
+  name: 'storybook-ui',
+  mode: 'production',
+
+  entry,
+  output: {
+    path: out,
+    filename: '[name]_dll.js',
+    library: '[name]_dll',
+  },
+  externals: provided,
+
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        use: [
+          {
+            loader: 'babel-loader',
```

Comment:
look like there is no need for js rules since you are pointing entries to the `dist`